api/desecapi/views.py

```diff
@@ -41,6 +46,52 @@ def get_client_ip(request):
     return request.META.get('REMOTE_ADDR')
 
 
+class TokenCreateView(djoser.views.TokenCreateView):
+
+    def _action(self, serializer):
+        user = serializer.user
+        token = Token(user=user)
```

Comment:
Elsewhere I suggested to have a default token name (because the migration assigns the name `'default'` to existing tokens). Seeing this line now, I agree that a token created upon regular login should have empty name, or name `'login'` or something similar.